plugins/oracle/identity/usersMfaEnabled.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     more_info: 'User accounts should have an MFA device setup to enable two-factor authentication.',
     link: 'https://docs.cloud.oracle.com/iaas/Content/Identity/Tasks/usingmfa.htm',
     recommended_action: 'Enable an MFA device for the user account.',
-    apis: ['user:list'],
+    apis: ['user:list', 'apiKey:list'],
```

Comment:
Do we need the second api call, are we using it any where

Reply:
Yes we are